frontend/src/Components/Endpoint/Endpoint.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import Button from "plaid-threads/Button";
+import Note from "plaid-threads/Note";
+
+import Table from "../Table/Table";
+import Error from "../Error/Error";
+
+import {
+  DataItem,
+  Categories,
+  ErrorDataItem,
+  Data,
+} from "../../Utilities/dataUtilities";
```

Comment:
unless we have multiple utilities files to put in this folder, this can probably live in just `src/dataUtilities.ts`

Reply:
done